### PR TITLE
Add ISRA page

### LINKED ISSUE #18: Add an ISRA page
For reference look at the issue #17

### PR DESCRIPTION
**Pull request by himanshuvarandani**
- [x] Closes #18 
---

## PR description

Add isra page and update link in header.

### CODE REVIEW

File: components/Isra.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          with thorough mentoring, the ISRA provides a closer look, helping the
+          aspirants from being familiarized with the research milieu of USA thereby
+          boosting their application according to their training.
+        </p>
+      </div>
+    </div>
+  )
+}
+
+export default Isra
```
Comment (pnijhara) [inline components/Isra.js:33]: leave an extra line.

File: components/Header.js [hunk 2]
```diff
@@ -158,11 +152,9 @@ export default function Header() {
               </div>
               <div className={`group h-full hover:bg-blue-850 ${router.pathname == '/alumni' && "bg-blue-850"}`}>
                 <div className="md:flex md:flex-col md:justify-center md:h-full">
-                  <Link href="">
-                    <a className="py-4 px-2 text-white text-sm">
-                      ALUMNI
-                    </a>
-                  </Link>
+                  <a className="py-4 px-2 text-white text-sm cursor-pointer">
+                    ALUMNI
```
Comment (pnijhara) [inline components/Header.js:156]: What is the reason for converting a link to an anchor tag?

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: components/Header.js (hunk 2 above)
Comment (himanshuvarandani) [reply components/Header.js:156]: Before this change, when we click on this, it changes the scrollable component at top.

Before click, page is shown as:
image

After click, it is forwarding to top:
image

Due to this reason, I think to remove href and make it text but Link tag needs an href so I removed that.

If this is not an issue, then I will add Link tag again.

Comment (pnijhara) [reply components/Header.js:156]: Got it!
Thank you for this. I was about to open a separate issue for this.

**Review by pnijhara — APPROVED** [review]

**Pull request merged**